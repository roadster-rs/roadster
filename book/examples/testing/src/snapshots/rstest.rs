use insta::assert_snapshot;
use roadster::testing::snapshot::TestCase;
use rstest::{fixture, rstest};

// File path: <crate_name>/src/snapshots/rstest.rs

#[fixture]
fn case() -> TestCase {
    Default::default()
}

#[rstest]
#[case(1)]
#[case(2)]
#[case::case_name(3)]
fn normalize_dynamic_uuid(_case: TestCase, #[case] value: u32) {
    assert_snapshot!(value);
}
