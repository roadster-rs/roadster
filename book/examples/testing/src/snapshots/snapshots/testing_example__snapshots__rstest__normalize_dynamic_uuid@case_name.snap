---
source: book/examples/testing/src/snapshots/rstest.rs
expression: value
---
3
